fabric-starter-client.js

```diff
@@ -119,15 +119,22 @@ class FabricStarterClient {
   }
 
   async invoke(channelId, chaincodeId, fcn, args, targets, waitForTransactionEvent) {
+    let peers = [];
     const channel = await this.getChannel(channelId);
+    let peer = channel.getChannelPeers();
+    for (let i = 0; i < targets.length; i++) {
```

Comment:
can you use array method [find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)?